planning/behavior_path_planner/autoware_behavior_path_planner_common/src/utils/path_utils.cpp

```diff
@@ -471,14 +471,15 @@ std::vector<Pose> interpolatePose(
 {
   std::vector<Pose> interpolated_poses{};  // output
 
-  const std::vector<double> base_s{
-    0, autoware::universe_utils::calcDistance2d(start_pose.position, end_pose.position)};
+  const double distance =
+    autoware::universe_utils::calcDistance2d(start_pose.position, end_pose.position);
+  const std::vector<double> base_s{0, distance};
```

Comment:
I think using an `int` as the 1st argument here could cause the wrong constructor to be called (number 3 in https://en.cppreference.com/w/cpp/container/vector/vector).
```suggestion
  const std::vector<double> base_s{0.0, distance};
```
Sorry for the annoying comments :sweat_smile:

Reply:
Agreed, I'll fix it right now.